app/controllers/obtain_annotations_with_callback_jobs_controller.rb

```diff
@@ -0,0 +1,85 @@
+class ObtainAnnotationsWithCallbackJobsController < ApplicationController
+  before_action :authenticate_root_user!, only: :new
+
+  def new
+    @project = Project.editable(current_user).find_by_name(params[:project_id])
+  end
+
+  def create
+    project = Project.editable(current_user).find_by_name(params[:project_id])
+    raise "Could not find the project: #{params[:project_id]}." unless project.present?
+    raise "Up to 10 jobs can be registered per a project. Please clean your jobs page." unless project.jobs.count < 10
+
+    # to determine the annotator
+    annotator = Annotator.find(params[:annotator])
+
+    # to determine the options
+    options = {}
+    options[:mode] = params[:mode]
+    options[:prefix] = annotator.name
```

Comment:
次のように書いても良いかもしれません。

```suggestion
    options = {
      mode: params[:mode],
      prefix: annotator.name
    }
```

好みの問題です。

Reply:
提案いただいたコードの方がいいと思ったのでコミットさせていただきました :pray:
https://github.com/pubannotation/pubannotation/pull/117/commits/459a2b52247747a84c9ca7d127376be930f5742c